ereuse_devicehub/resources/action/views.py

```diff
@@ -21,12 +21,18 @@
 SUPPORTED_WORKBENCH = StrictVersion('11.0')
 
 
-def save_json(req_json, tmp_snapshots):
+def save_json(req_json, tmp_snapshots, user):
     """
     This function allow save a snapshot in json format un a TMP_SNAPSHOTS directory
     The file need to be saved with one name format with the stamptime and uuid joins
     """
-    name_file = "{uuid}_{time}.json".format(uuid=req_json.get('uuid', ''), time=int(time()))
+    uuid = req_json.get('uuid', '')
+    now = datetime.now()
+    year = now.year
+    month = now.month
+    day = now.day
+
+    name_file = f"{uuid}_{user}_{year}-{month}-{day}.json"
```

Comment:
Now I have realized that to have ordered by date when the snapshot is uploaded, the name of the json would have to have the date first, I would even add the {hour}-{minutes}.

Then I propose:
**name_file = f"{year}-{month}-{day}-{hour}-{minutes}_{user}_{uuid}.json"**

Reply:
Done